.circleci/config.yml

```diff
@@ -1,15 +1,19 @@
 version: 2.1
 
 orbs:
-  gem: doximity/gem-publisher@0
+  gem: doximity/gem-publisher@dev:3673d17
```

Comment:
is this  a permanent tag?

Reply:
No, I'm reverting this to `0` now that we released an update to gem-publisher.